InvenTree/part/api.py

```diff
@@ -405,13 +405,6 @@ def filter_queryset(self, queryset):
             except (ValueError, Part.DoesNotExist):
                 pass
 
-        # Filter by latest part creation date
-        latest_parts = params.get('latest_parts', None)
-
-        if latest_parts is not None:
-            # Get the last 5 created parts
-            queryset = queryset.order_by('-creation_date')[:5]
```

Comment:
@eeintech the [django DRF framework](https://www.django-rest-framework.org/) already provides queryset ordering functionality, you just need to add 'creation_date' to the fields available for ordering

Reply:
@SchrodingersGat Ha right, this looks much more elegant. Thanks for the improvements!